### PR TITLE
[chromecast] Added support for next command

### PR DESCRIPTION
**Pull request by boc-tothefuture**
[chromecast] Next Command Support

Add support for next command to chromecast binding, this is implemented by seeking to the end of the current track as there is no native next command.  This is the same implementation for next that the python library uses for chromecast, which is used by home assistant to achieve this functionality.

### CODE REVIEW

File: bundles/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/internal/ChromecastCommander.java [hunk 1]
```diff
@@ -135,10 +135,19 @@ private void handlePlayUri(Command command) {
     private void handleControl(final Command command) {
         try {
             if (command instanceof NextPreviousType) {
-                // I can't find a way to control next/previous from the API. The Google app doesn't seem to
-                // allow it either, so I suspect there isn't a way.
-                logger.info("{} command not yet implemented", command);
-                return;
+                // Next is implemented by seeking to the end of the current media
+                if (command == NextPreviousType.NEXT) {
```
Comment (lolodomo) [inline bundles/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/internal/ChromecastCommander.java:139]: Maybe it has to be moved beliw, after checking if app is running?

**Comment by lolodomo** [comment]
@boc-tothefuture : no time to answer to my comment ?

File: bundles/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/internal/ChromecastCommander.java (hunk 1 above)
Comment (boc-tothefuture) [reply bundles/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/internal/ChromecastCommander.java]: Good catch - moved.

**Comment by boc-tothefuture** [comment]
> @boc-tothefuture : no time to answer to my comment ?

Didn't see the notification. Sorry.

**Review by lolodomo — APPROVED** [review]
LGTM

**Comment by lolodomo** [comment]
Please sign property your last commit.

**Comment by lolodomo** [comment]
Let's make an exception as the initial PR was correctly signed.

**Comment by boc-tothefuture** [comment]
@lolodomo sorry for the delay.. For some reason I am not seeing notifications from this repo, and only this repo.  I will recheck my setting.

**Pull request merged**